### PR TITLE
mir.examples: add means of columns

### PR DESCRIPTION
**Pull request by wilzbach**
moved means of columns over to mir

See https://github.com/DlangScience/examples

We should probably wait for the Windows CI Feedback ...

### CODE REVIEW

File: examples/means_of_columns.d [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        .map!(r => sum(r, 0L) / cast(double) r.length)
+        .array;
+}
+
+void main() {
+    data = 100_000.iota.array;
+    auto r = benchmark!(f0)(testCount);
+    auto f0Result = to!Duration(r[0] / testCount);
+    f0Result.writeln;
+}
```
Comment (9il) [inline examples/means_of_columns.d:49]: LL please

Comment (wilzbach) [inline examples/means_of_columns.d:49]: LL?

Comment (9il) [inline examples/means_of_columns.d:49]: Last Line

**Comment by wilzbach** [comment]
> We should probably wait for the Windows CI Feedback ...

wasn't joking - ok let's hope for the best ;-)

**Comment by 9il** [comment]
> wasn't joking - ok let's hope for the best ;-)

This does not contains other packages and is very simple

**Comment by wilzbach** [comment]
> is very simple

his famous last words :D - ok!

**Pull request merged**